object.go

```diff
@@ -281,22 +281,60 @@ func (js *js) DeleteObjectStore(bucket string) error {
 	return js.DeleteStream(stream)
 }
 
-func sanitizeName(name string) string {
-	stream := strings.ReplaceAll(name, ".", "_")
-	return strings.ReplaceAll(stream, " ", "_")
+func encodeName(name string) string {
+	return base64.URLEncoding.EncodeToString([]byte(name));
+}
+
+// toDefaultMetaSubject and toPublishMetaSubject will diverge once there is account support
+func toDefaultMetaSubject(obs *obs, name string) string {
```

Comment:
Should this be a func on (*obs)?

Reply:
That's where I don't know go well enough